### ISSUE TITLE
feat: after migration to jetty 11 (jakarta) arquillian server side tests are not running -> migrate to 1.7.0.Alpha10

### ISSUE THREAD

**Issue by yuriyz** [JanssenProject/jans#1247]
### Description

feat: after migration to jetty 11 (jakarta) arquillian server side tests are not running

![image](https://user-images.githubusercontent.com/2013070/164737404-0ef08d68-90ab-43e9-8a9b-b4cf911398a6.png)

**Comment by Milton-Ch** [comment]
I was testing multiple configuration about this issue, latest version of Arquillian with Jetty and TestNG didn't work, looks like there is some compatibility issue, I was able to get it working with JUnit, however with TestNG there is a problem processing the test, I got many errors like this:
```
Caused by: org.jboss.weld.exceptions.AmbiguousResolutionException: WELD-001318: Cannot resolve an ambiguous dependency between: 
  - Managed Bean [class io.jans.as.server.service.ResteasyInitializer] with qualifiers [@Any @Default],
  - Managed Bean [class io.jans.as.server.service.ResteasyInitializer] with qualifiers [@Any @Default]
```
I tried to review why we have two times same bean in Weld, however there was no solution, I tried to put this manually as well, but nothing worked.

Passing this ticket to @yurem since he has more experience on these tools.

BTW, latest version of arquillian container with Jetty 11 was released recently, I was able to get it working with JUnit, but didn't work with TestNG, configurations for both tools are different.
